heron/api/src/java/org/apache/heron/api/serializer/JavaSerializer.java

```diff
@@ -40,7 +40,7 @@ public void initialize(Map<String, Object> config) {
       oos.writeObject(object);
       oos.flush();
     } catch (IOException e) {
-      throw new RuntimeException(e);
+      throw new RuntimeException("NotSerializable object is: " + object.toString(), e);
```

Comment:
Suggest: "Failed to serialize object: " + object.toString()

Reply:
updated